composer.json

```diff
@@ -20,18 +20,19 @@
         "ruflin/elastica": "^3.0",
         "symfony/symfony": "~2.8|~3.0",
         "jms/serializer-bundle": "^1.1",
-        "oldsound/rabbitmq-bundle": "^1.8"
+        "oldsound/rabbitmq-bundle": "^1.8",
+        "laravel/framework": "^5.1"
     },
 
     "autoload": {
-        "psr-0": {
-            "Tolerance\\": "src/"
+        "psr-4": {
+            "": "src/"
```

Comment:
For `autoload` it doesn't change anything, but it's needed for `autoload-dev` to include files like `tests/PHPUnit/IlluminateTestCase.php`, I thought we should just keep only one. Doesn't change anything in the end it's more a matter of preference so up to you